lib/orders/submit_order.js

```diff
@@ -38,11 +38,17 @@ module.exports = async (strategyState = {}, order = {}) => {
   return new Promise((resolve, reject) => {
     submitOrder(ws, order).then((filledOrder) => {
       debug('order submitted')
+
       perfManager.addOrder({
         amount: new BigNumber(filledOrder.amount),
         price: new BigNumber(filledOrder.price)
       })
-      resolve(filledOrder)
+
+      resolve({
```

Comment:
makes the listener redundant

Reply:
I had the impression that the promise was not resolved, but I did new test and you're right

Reply:
58cc85f1ae46f0982bcf65ebe1a8726d6fc421bc